<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportExportDatabaseDialog</class>
 <widget class="QDialog" name="ImportExportDatabaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export / Import Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Export</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushExportToFile">
        <property name="text">
         <string>Export to file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushExportToGDrive">
        <property name="text">
         <string>Export to Google Drive</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Import</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="pushImportFile">
        <property name="text">
         <string>Import from file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushImportGDrive">
        <property name="text">
         <string>Import from Google Drive</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
